src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -103,32 +172,45 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
     
     config = loadConfig(config_filename)
     num_tests = len(config)
-    test_pass_threshold = 0.95 #percentage of passed test expected
+    test_pass_threshold = 0.999 #percentage of passed test expected
     num_passed_tests = 0.0
```

Comment:
This should really be an integer (here and below). You're never going to pass half a test.